src/main/scala/fs2/Process1.scala

```diff
@@ -104,6 +104,12 @@ object process1 {
   def scan1[F[_],I](f: (I, I) => I)(implicit F: NotNothing[F]): Handle[F,I] => Pull[F,I,Handle[F,I]] =
     Pull.receive1 { case o #: h => scan(o)(f).apply(h) }
 
+  /** Writes the sum of all input elements, or zero if the input is empty. */
+  def sum[F[_],I:Numeric](implicit F: NotNothing[F]): Handle[F,I] => Pull[F,I,Handle[F,I]] = {
```

Comment:
I don't think util.Monoid exists (yet). Also, AFAIK, these are supposed to be internal typeclasses, not to be part of the external API.

I appreciate that this uses up the method "sum" that could be used with typeclasses from cats or scalaz in bridging APIs; but this is (approximately) the signature from <= 0.8...